### PR TITLE
[MIG] account_invoice_change_currency: Migration to 15.0

### PR DESCRIPTION
**Pull request by rolandojduartem**

### CODE REVIEW

**Comment by rolandojduartem** [comment]
Hi, @luisg123v, @moylop260 and @hugho-ad, please, may someone review this migration? T#52382

red tests are related to account_invoice_fiscal_position_update

cc [@name1984](https://github.com/name1984) @desdelinux)

File: account_invoice_change_currency/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Invoice - Change Currency",
+    "version": "15.0.1.0.0",
+    "category": "Accounting & Finance",
+    "summary": "Allows to change currency of Invoice by wizard",
+    "author": "Vauxoo, Komit Consulting, Odoo Community Association (OCA)",
+    "maintainers": ["luisg123v"],
```
Comment (hugho-ad) [inline account_invoice_change_currency/__manifest__.py]: add your it yourself

File: account_invoice_change_currency/tests/test_account_invoice_change_currency.py [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+    def setUp(self):
+        super().setUp()
+
+        self.precision = self.env["decimal.precision"].precision_get("Payment Terms")
+        res_users_account_manager = self.env.ref("account.group_account_manager")
+        self.manager = (
+            self.env["res.users"]
+            .with_context(no_reset_password=True)
+            .create(
+                dict(
```
Comment (hugho-ad) [inline account_invoice_change_currency/tests/test_account_invoice_change_currency.py:18]: why not use the {} definition?

Comment (moylop260) [reply account_invoice_change_currency/tests/test_account_invoice_change_currency.py:18]: @hugho-ad be aware if original is as it so any change related could be out of only migration

File: account_invoice_change_currency/__manifest__.py (hunk 1 above)
Comment (rolandojduartem) [reply account_invoice_change_currency/__manifest__.py]: Done!

File: account_invoice_change_currency/tests/test_account_invoice_change_currency.py (hunk 2 above)
Comment (rolandojduartem) [reply account_invoice_change_currency/tests/test_account_invoice_change_currency.py:18]: I think this out of migration process before v14.0

**Comment by rolandojduartem** [comment]
Hi, @luisg123v, @moylop260 and @hugho-ad, please, may you review this migration? T#52382

red tests are related to account_invoice_fiscal_position_update

cc @name1984 @desdelinux

**Review by luisg123v — APPROVED** [review]

**Review by hugho-ad — APPROVED** [review]

**Comment by moylop260** [comment]
Could you create a new issue related to the red CI for `account_invoice_fiscal_position_update` module, please?

**Comment by pedrobaeza** [comment]
The red CI is due to upstream changes, as the last time. You can collaborate as well to fix it. I think this time it can be something with explicitly calling action_post that I have seen around.

**Pull request merged**